lib/query/query_payments.rb

```diff
@@ -366,7 +366,7 @@ def self.get_dedication_or_empty(*path)
   end
 
   def self.export_selects
-    ["to_char(payments.date::timestamptz, 'YYYY-MM-DD HH24:MI:SS TZ') AS date",
+    ["to_char(payments.date::timestamptz at time zone nonprofits.timezone, 'YYYY-MM-DD HH24:MI:SS TZ') AS date",
```

Comment:
Don't you have to coerce to UTC if the `nonprofits.timezone` is null?

Reply:
Yes, I forgot to do it here, thanks!